build-tools/xaprepare/xaprepare/Main.cs

```diff
@@ -101,6 +101,7 @@ static async Task<int> Run (string[] args)
 				{"a|enable-all", "Enable preparation of all the supported targets, ABIs etc", v => parsedOptions.EnableAll = true},
 				{"b|bundle-path=", "Full path to the {{DIRECTORY}} where Xamarin.Android bundle can be found (excluding the file name)", v => parsedOptions.XABundlePath = v?.Trim ()},
 				{"copy-bundle-to=", "Full path to the {{DIRECTORY}} where downloaded bundle should be copied (excluding the file name)", v => parsedOptions.XABundleCopyDir = v?.Trim ()},
+				{"refresh=", "[sdk,ndk] Comma separated list of components which should be reinstalled.", v => parsedOptions.RefreshList = ParseRefreshableComponents (v?.Trim ())},
```

Comment:
Do we need a way to trigger this when you queue a build on Azure DevOps? Adding a boolean variable at queue time?

Reply:
I was planning to use this in the APK test PR to replace this hack - https://github.com/xamarin/xamarin-android/pull/3042/files#diff-9268c11f675853b241d743a346444d86R263.

I think it makes sense to conditionally add this to other build stages though based on the value of a queue time variable, I'll take a look at adding that.